src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/SphKerrSchild.hpp

```diff
@@ -35,6 +35,291 @@ class not_null;
 namespace gr {
 namespace Solutions {
 
+/*!
+ * \brief Kerr black hole in Spherical Kerr-Schild coordinates
+ *
+ * ## Introduction
+ *
+ * Given a Kerr-Schild (KS) black hole system, we denote the coordinate system
+ * using \f$\{t,x,y,z\}\f$. In the transformed system, Spherical Kerr-Schild
+ * (SphKS), we denote the coordinate system using
+ * \f$\{t,\bar{x},\bar{y},\bar{z}\}\f$. Further, when considering indexed
+ * objects, we will use Greek and Latin indices with the standard convention
+ * (\f$\mu=0,1,2,3\f$ and \f$i=1,2,3\f$ respectively), but we will use a bar to
+ * denote that a given index is in reference to the SphKS coordinate system
+ * (i.e. \f$i\f$ vs. \f$\bar{\imath}\f$ and \f$\mu\f$ vs. \f$\bar{\mu}\f$).
+ *
+ * ## Spin in the z direction
+ *
+ * ### The Transformation
+ *
+ * The Boyer-Lindquist radius for KS with its spin in the \f$z\f$ direction is
+ * defined by
+ * \f{align}{
```

Comment:
would you mind adding an blank line:
```cpp
 * defined by
 *
 * \f{align}{
```
and also after the equations? Clang-format sometimes squashes things onto one line and then unintentionally breaking documentation